website/integrations/services/nextcloud/index.md

```diff
@@ -129,15 +139,15 @@ Add a new provider using the `+` button and set the following values:
     :::
 -   Scope: `email`, `profile` (you can safely omit `openid` if you prefer)
 -   Attribute mappings:
-    -   User ID mapping: sub
+    -   User ID mapping: sub (or `user_id` if you need to connect to already existing Nextcloud user)
     -   Display name mapping: name
     -   Email mapping: email
     -   Quota mapping: quota (leave empty if you have skipped the [custom profile scope](#custom-profile-scope) section)
     -   Groups mapping: groups (leave empty if you have skipped the [custom profile scope](#custom-profile-scope) section)
         :::tip
         You need to enable the "Use group provisioning" checkmark to be able to write to this field
         :::
--   Use unique user ID: If you only have one provider you can uncheck this if you prefer.
+-   Use unique user ID: If you only have one provider you can uncheck this if you prefer (This will affect your Federated Cloud ID, under _Personal settings_ -> _Sharing_ -> _Federated Cloud_).
```

Comment:
```suggestion
-   Use unique user ID: If you only have one provider you can deselect this if you prefer (This will affect your Federated Cloud ID, under _Personal settings_ -> _Sharing_ -> _Federated Cloud_).
```

Reply:
@Fymyte is it worth explaining exactly HOW deselecting this field will affect the ID?

Reply:
I am not 100% sure. If I am not mistaken, if you tick the box the user id will be a random UUID generated by nextcloud, so you will have something like `0101830273902827722@nextcloud.instance`.
If the box is not ticked, the id will be the one of the `user_id` set by Authentik, so Authentik's user uuid, or `nextcloud_user_id` if set.
This would allow, for example, to get a federated cloud id `Fymyte@nextcloud.instance` if you set the `nextcloud_user_id`.

Reply:
I tried to explain it better. Can you re-check ?

Reply:
Hi @Fymyte sorry I was out on vacation. Thanks for that additional info on line 150.

Reply:
NP thanks for the feedbacks